_redirects.testing

```diff
@@ -1,2 +1,2 @@
-/api/*  https://api-testing.frost.po.et/:splat  200
+/api/*  https://frost-api.staging.po.et/:splat  200
```

Comment:
@WesleyCharlesBlake 
what do you think  if we change the file name to `_redirects.staging` or create a new one?

Reply:
Could do. But I'm not sure on how we would get netlify to create an environment from that? If all I need to do is is create a new redirects file then I will do so.